<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>360</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>780</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Access Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Active Status</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_un"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_pw"/>
  </widget>
  <widget class="QComboBox" name="comboBox_al">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>LEVEL-1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LEVEL-2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DEMO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NO ACCESS</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_ac">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>INACTIVE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ACTIVE</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
